pkg/rancher-desktop/window/index.ts

```diff
@@ -479,14 +479,16 @@ export function openDialog(id: string, opts?: Electron.BrowserWindowConstructorO
     return window;
   }
 
-  Shortcuts.register(
-    window,
-    { key: 'Escape' },
-    () => {
-      window.close();
-    },
-    'Close dialog',
-  );
+  if (!opts || opts.closable) {
```

Comment:
This appears to break the ability to press escape to close the Reset Kubernetes and Factory Reset dialogs

Reply:
This is a bug.